src/domain/event/utils.ts

```diff
@@ -65,6 +65,15 @@ export const getEventStartTimeStr = (
 export const isEventFree = (event: EventFieldsFragment): boolean =>
   Boolean(event.offers.find((item) => item.isFree)?.isFree);
 
+/**
+ * Check if the additional information is requested as mandatory.
+ * @param eventData
+ * @return {boolean}
+ */
+export const isMandatoryAdditionalInformationRequired = (
+  event: EventFieldsFragment
+): boolean => Boolean(event.pEvent?.mandatoryAdditionalInformation);
```

Comment:
Seems little bit strange to me that this is abstracted to another function, is it really necessary? 🤔

Reply:
it's not necessary, but it was consistent, since the isFree had similar approach. Also, I feel that pEvent.mandatoryAdditionalInformation is a little bad name for a boolean field (it looks a text field to me 🧐), even now when we are using typed variables. Sure it adds a trivial method in a code base, so in that sense, should we leave it out. 

So which approach will we choose?

Reply:
I removed it because it was used only here

Reply:
Ofc there is no right answer but to me I think helper function wasn't needed